### ISSUE TITLE
implement place path card test and logic 5

### ISSUE THREAD

**Issue by kayac-chang** [Game-as-a-Service/saboteur-remix#29]
```
given:
  a board with
    - a start card at position (0, 0)
when:
  place
    - path card [connected cross] at position (0, 1)
then:
  - should return
    - path card has been placed event with
      - path card [connected cross] at position (0, 1)
  - event source should includes
    - path card has been placed event with
      - path card [connected cross] at position (0, 1)
```

**Comment by MatthewNoah4132** [comment]
補圖

![image](https://user-images.githubusercontent.com/122686825/233683207-61a35ab3-ab14-4de7-addc-fd942f966fbf.png)

**Comment by kayac-chang** [comment]
closed #35
